pack/fhv/fhvc.json

```diff
@@ -283,6 +283,174 @@
+    "double_sided": true,
+    "encounter_code": "day_of_the_feast",
+    "encounter_position": 2,
+    "faction_code": "mythos",
+    "flavor": "A beautiful sheen hangs over everything, shimmering in the air around you.",
+    "illustrator": "Rob Laskey",
+    "name": "Dawn of the Final Day",
+    "pack_code": "fhvc",
+    "position": 187,
+    "quantity": 1,
+    "stage": 1,
+    "text": "<b>Objective</b> – Talk to the locals and prepare yourself for the Feast.",
+    "type_code": "act"
+  },
+  {
+    "back_name": "The Sleep",
+    "back_text": "<blockquote><i>Glowing spores hang in the air around the effigy, lighting up the mirrors and bathing the festival in a faerie glow. You breathe in and feel your mouth and throat buzz with a light, tingling sensation. The last thing you hear as you pas out is the sound of lilting flutes and tambourines.</i></blockquote>\nEach investigator is defeated and suffers 1 mental or 1 physical trauma.",
+    "code": "10688",
+    "doom": 6,
+    "double_sided": true,
+    "encounter_code": "day_of_the_feast",
+    "encounter_position": 3,
+    "faction_code": "mythos",
+    "flavor": "Revelers dance to raucous music while others gorge themselves on rank platters of oily food. Visitors and locals alike collapse in fits of laughter while others claw at their eyes. Each of them weat a wild, gleeful smile.",
+    "illustrator": "Rob Laskey",
+    "name": "Around the Table",
+    "pack_code": "fhvc",
+    "position": 188,
+    "quantity": 1,
+    "stage": 2,
+    "text": "<b>Objective</b> – Confront Mother Rachel! If Mother Rachel is defeated: <b>(→R1)</b>.",
+    "type_code": "agenda"
+  },
+  {
+    "back_name": "Back in the Fold",
+    "back_text": "<blockquote><i>\"Well done,\" Mother Rachel says to you. Lights dance in the sky overhead as the last of the Vale residents are led dutifully to their place at the table. Some look at your with rueful expressions, rightfully betrayed. As you move to assume your place at the table, Mother Rachel steps in your way.\n\"It is not your place to sit with us. You are our guest of honor,\" she smiles. You feel a sharp blow to the back of your head, and then nothing.</i></blockquote>\nEach investigator resigns.",
+    "code": "10689",
+    "doom": 5,
+    "double_sided": true,
+    "encounter_code": "day_of_the_feast",
+    "encounter_position": 4,
+    "faction_code": "mythos",
+    "flavor": "You must gather Mother Rachel's flock, willing or not, to their place at the table.",
+    "illustrator": "Rob Laskey",
+    "name": "Lambs to the Slaughter",
+    "pack_code": "fhvc",
+    "position": 189,
+    "quantity": 1,
+    "stage": 3,
+    "text": "You cannot trigger [codex] codex entries.\n[reaction] After you evade a [[Resident]] enemy by 2 or more: Add them to the victory display.\n<b>Objective</b> – if 1 plus 1 [per_investigator] [[Resident]] cards are in the victory display, <b>(→R3)</b> <i>(page 60)</i>.",
+    "type_code": "agenda"
+  },
+  {
+    "code": "10692",
```

Comment:
Please move this card to its correct spot (after the next two cards) by card number.